dap_prinz_green_jobs/tests/test_industry_measures.py

```diff
@@ -67,7 +64,15 @@ def test_clean_unit_emissions_dict():
     assert clean_unit_emissions_dict({"10.2-3": 10}) == {"102": 10, "103": 10}
 
 
+IN_GITHUB_ACTIONS = os.getenv("GITHUB_ACTIONS") == "true"
+
+
+@pytest.mark.skipif(IN_GITHUB_ACTIONS, reason="Test doesn't work in Github Actions.")
```

Comment:
oooooh so cool

Reply:
lets see if it works! 
Better than no tests!

Reply:
hmmmm interesting:

```
ERROR dap_prinz_green_jobs/tests/test_industry_measures.py - PermissionError: Forbidden
ERROR dap_prinz_green_jobs/tests/test_occupation_measures.py - AttributeError: module 'numpy' has no attribute 'int'.
```

Reply:
:( well.. I thought it was easily fixed but I guess np.int is referenced in another package rather than something we did.. will look into it for another 5 mins!

Reply:
seems like the np.int issue is being fixed by sklearn https://github.com/scikit-optimize/scikit-optimize/issues/1171.. so I guess we could keep the github action as it is at the mo - and then hopefully at some point soon [the fix](https://github.com/scikit-optimize/scikit-optimize/pull/1187) is merged and the new version of sklearn works?

Reply:
ooooh interesting

Reply:
ok sounds like a plan - I'll update the action to not use conda/install faiss in the meantime